iroha_client_no_std/tests/check_health.rs

```diff
@@ -0,0 +1,26 @@
+#[cfg(test)]
+mod tests {
+    use async_std::task;
+    use iroha::{config::Configuration, maintenance::*, Iroha};
+    use iroha_client_no_std::{
+        client::Client, config::Configuration as ClientConfiguration, prelude::*,
+    };
+    use std::thread;
+    use tempfile::TempDir;
+
+    const CONFIGURATION_PATH: &str = "tests/test_config.json";
+
+    fn create_and_start_iroha() {
+        let temp_dir = TempDir::new().expect("Failed to create TempDir.");
+        let mut configuration =
+            Configuration::from_path(CONFIGURATION_PATH).expect("Failed to load configuration.");
+        configuration
+            .kura_configuration
+            .kura_block_store_path(temp_dir.path());
+        let iroha = Iroha::new(configuration);
+        task::block_on(iroha.start()).expect("Failed to start Iroha.");
+        //Prevents temp_dir from clean up untill the end of the tests.
+        #[allow(clippy::empty_loop)]
+        loop {}
+    }
+}
```

Comment:
These test doesn't needed for now - I removed maintenance functionality from current no-std